### ISSUE TITLE
Image compression Mime type exclusion list

### ISSUE THREAD

**Issue by sigma-technology** [elninotech/uppload#103]
**Is your feature request related to a problem? Please describe.**
We would like to be able to optimise all JPG files uploaded, but leave all PNG files alone (i.e. not convert them to a different format).  It's not possible to do this currently as specifying the `compression` property converts all images to JPG.

**Describe the solution you'd like**
To cater for different scenarios and requirements, I think the best solution is probably to have an 'exclusion list' of mime formats that should never be converted.  Something like:

    const uploader = new Uppload({
      compression: 0.8,
      compressionMime: "image/webp",
      compressionMimeExclude: "image/png"
    });

 Hopefully this would also maintain any transparency the PNG has.

**Describe alternatives you've considered**
There isn't really an alternative that I can think of.

**Comment by AnandChowdhary** [comment]
Instead of excluding, I'm following the convention of specifying MIME types that we've established. In this update, we've **removed** the `compressionMime` property and replaced it with 
`compressionFromMimes` and `compressionToMime`, which you can use like this:

```ts
const uploader = new Uppload({
  compression: 0.8,
  compressionFromMimes: ["image/jpeg", "image/webp"],
  compressionToMime: "image/webp"
});
```

This is available in `v2.7.0`.

**Comment by sigma-technology** [comment]
Excellent, thank you - this is indeed a better solution than the one I proposed.
